### DISCUSSION TITLE
In WSL2 Ubuntu, pasting an answer doesn't paste - why and workaround?

### DISCUSSION THREAD

**Discussion by ndemarco** [copier-org/copier#1881] [General]
When providing answers in a WSL2 session running Ubuntu 22 & 24 via Windows Terminal, right clicking normally pastes values. When answering a Copier template, right clicking doesn't paste. I also tried CTRL-V.

How can I paste values into a Copier session within a WSL2 session?

**Comment by pawamoy** [comment]
Try `ctrl-shift-v`. Different terminal emulators react to different keyboard/mouse inputs 🙂 Though here you're in Windows Terminal so not sure what program actually intercepts the inputs.

**Reply by ndemarco** [reply]
🤦
Today, I'm being blocked by the little things. 🐌

**Reply by Mark-Booth** [reply]
I'm seeing the same in a Gnome Terminal too (v3.28.2 on RHEL 8).

While `copier` is running, it completely disables selection, and middle-click paste, and since I can't select, I can't even use the Copy option on the context (right-click) menu.

As soon as `copier` finishes running, selection and middle-click paste start working again.
